cmdstanpy/cmdstan_args.py

```diff
@@ -277,6 +277,38 @@ def compose(self, idx: int, cmd: str) -> str:
         return cmd
 
 
+class GenerateQuantitiesArgs(object):
+    """Arguments for the generated quantities block."""
+    def __init__(
+        self,
+        fitted_params_file: str = None,
+    ) -> None:
+        """Initialize object."""
+        self.fitted_params_file = fitted_params_file
+
+    def validate(self,chains=None) -> None:
+        """
+        Check arguments correctness and consistency.
+        * file for fitted_params_file exists
+        """
+        if self.fitted_params_file is not None:
+            if not os.path.exists(self.fitted_params_file):
+                raise ValueError(
+                    'Invalid path for fitted_params_file: {}'.format(
+                        self.fitted_params_file
+                        )
+                )
+
+    def compose(self, idx: int, cmd: str) -> str:
+        """
+        Compose CmdStan command for method-specific non-default arguments.
+        """
+        if self.fitted_params_file is not None:
+            cmd = cmd + ' method=generate_quantities'
+            cmd = '{} fitted_params={}'.format(cmd, self.fitted_params_file)
```

Comment:
instead of making a string, could this gather `cmd` list, so arguments (paths) with spaces is handled correctly?

Reply:
In the validate function, we use 'os.path.exists' (which handles file paths with spaces) to validate if the file exists. So the file that will get passed in here will be a valid file.

Reply:
Ok, I noticed, that do_sample functions still uses `.split` which is suboptimal (bug), so this can be left as is, and let's fix do_sample and all the cmd in other PR.